From 0520dcce727b639788c100575abfc06d9a149039 Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 21:07:48 +0800
Subject: [PATCH 0123/5242] MLK-9638-2 usb: otg: test: add otg_hnp_reqd flag
 in struct otg_fsm

commit  36dae5e90112eb7c945be948f1f018b80f72e676 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds flag otg_hnp_reqd in struct otg_fsm to indicate A device
as host wants to start HNP sequence.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 92382e06c6902731b05dcbb0db4f0ebfa486969f)
(cherry picked from commit 8e3f26a58f995711a81b401af26fbe5ed12e17f4)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/usb/otg-fsm.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 718d93a..5524d01 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -179,6 +179,7 @@ struct otg_fsm {
 	int tst_maint;
 	int otg_vbus_off;
 	int otg_srp_reqd;
+	int otg_hnp_reqd;
 
 	/* Informative variables. All unused as of now */
 	int a_bus_drop_inf;
-- 
1.7.9.5

